libxmf/XmfVpxDecoder.c

```diff
@@ -0,0 +1,106 @@
+#include "XmfVpxDecoder.h"
+#include <stdlib.h>
+#include <string.h>
+#include <vpx/vp8dx.h>
+#include <vpx/vpx_decoder.h>
+#include "XmfVpxImage.h"
+
+// Internal decoder structure
+struct xmf_vpx_decoder
+{
+    vpx_codec_ctx_t codec;
+    XmfVpxDecoderError lastError;
+};
+
+XmfVpXDecoder *XmfVpxDecoder_Create(XmfVpxDecoderConfig cfg)
+{
+
+    XmfVpXDecoder *decoder = (XmfVpXDecoder *)malloc(sizeof(XmfVpXDecoder));
+    if (!decoder)
+    {
+        return NULL;
+    }
+
+    memset(decoder, 0, sizeof(XmfVpXDecoder));
```

Comment:
nit: why not use `calloc` instead of `malloc` and then immediately 0 the memory?

Reply:
updated